src/ray/raylet/worker_pool_test.cc

```diff
@@ -63,9 +63,11 @@ class WorkerPoolTest : public ::testing::Test {
 static inline TaskSpecification ExampleTaskSpec(
     const ActorID actor_id = ActorID::nil(),
     const Language &language = Language::PYTHON) {
+  std::vector<std::string> function_descriptor(3);
+  function_descriptor.push_back(UniqueID::from_random().binary());
```

Comment:
we can simply add 3 empty strings to the fd, no need to add random unique id, right?
(comment also applies to a few similar places above.)